src/components/wizard/wizard.component.spec.ts

```diff
@@ -249,3 +246,126 @@ describe('Wizard with validator', () => {
             .innerText.trim();
     }
 });
+
+interface WizardStep {
+    header: string;
+    content: string;
+    visited?: boolean;
+    valid?: boolean;
+}
+
+@Component({
+    selector: 'wizard-visited-change-test-app',
```

Comment:
Synchronise this with the marquee wizard tests above. Since you have introduced `resetVisitedOnValidationError` into the standard wizard, those tests need to be done here as well.